components/multi-platform-controller/production-downstream/host-config.yaml

```diff
@@ -60,7 +60,7 @@ data:
   dynamic.linux-arm64.aws-secret: aws-account
   dynamic.linux-arm64.ssh-secret: aws-ssh-key
   dynamic.linux-arm64.security-group-id: sg-0903aedd465be979e
-  dynamic.linux-arm64.max-instances: "50"
+  dynamic.linux-arm64.max-instances: "250"
```

Comment:
we should increase amd64 too

Reply:
why? it is a local platform.

Reply:
`amd64` isn't local, it is a provisioned VM. The local platforms are defined above. `x86_64` is an example of a local one.